test/libsolidity/SolidityNatspecJSON.cpp

```diff
@@ -328,6 +328,54 @@ BOOST_AUTO_TEST_CASE(public_state_variable)
 	checkNatspec(sourceCode, "test", userDoc, true);
 }
 
+BOOST_AUTO_TEST_CASE(public_state_variable_struct)
+{
+	char const* sourceCode = R"(
+		contract Bank {
+			struct Coin {
+				string obverseGraphicURL;
+				string reverseGraphicURL;
+			}
+
+			/// @notice Get the n-th coin I own
+			/// @return obverseGraphicURL Front pic
+			/// @return reverseGraphicURL Back pic
```

Comment:
Could you also add a test for duplicate `@return` for one parameter while another is missing? I.e.
```
			/// @return obverseGraphicURL Front pic
			/// @return obverseGraphicURL Front pic
```

Reply:
sure